config/federation/grafana/dashboards/Pipeline_PT.json

```diff
@@ -0,0 +1,2423 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
```

Comment:
Oh, these Grafana annotations have been causing some weird behavior. Please change "enable" to "false".